src/Nethermind/Nethermind.Blockchain.Test/KnownChainSizesTests.cs

```diff
@@ -14,7 +14,7 @@ public class KnownChainSizesTests
     public void Update_known_chain_sizes()
     {
         // Pruning size have to be updated frequently
-        ChainSizes.CreateChainSizeInfo(BlockchainIds.Mainnet).PruningSize.Should().BeLessThan(200.GB());
+        ChainSizes.CreateChainSizeInfo(BlockchainIds.Mainnet).PruningSize.Should().BeLessThan(250.GB());
```

Comment:
imo +25% is too big for hotfix

Reply:
10gb should be fine

Reply:
resolved in [f379e33](https://github.com/NethermindEth/nethermind/pull/6087/commits/f379e33bedba522f3fef094b2bba0c3a238a9e15)